templates/index.html

```diff
@@ -119,7 +120,7 @@ <h2>Schedule</h2>
                         <td>Sprints and Community Meetings</td>
                     </tr>
                 </table>
-                <a href="https://cfp.matrix.org" class="cta">Propose a Talk</a>
+                <a href="/schedule" class="cta">View the schedule</a>
```

Comment:
suggest "View Schedule" for consistency w the other button. also, thank you and sorry for not seeing this sooner!